doc-content/enterprise-only/openshift/environment-immutable-modify-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='environment-immutable-modify-proc']
 = Modifying the template for deploying an immutable {KIE_SERVER} using S2I
 
-By default, the template for deploying an immutable server using S2I creates a separate PostgreSQL pod to provide the database server for each replicable {KIE_SERVER}. If you prefer to use MySQL or to use an external server (outside the OpenShift project), you must modify the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver.yaml` template before deploying the server.
+By default, the template for deploying an immutable server using S2I creates a separate PostgreSQL pod to provide the database server for each replicable {KIE_SERVER}. If you prefer to use MySQL or to use an external server (outside the OpenShift project), you must modify the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver.yaml` or `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver-amq.yaml` template file before deploying the server.
```

Comment:
No need to repeat "to use" twice.
Suggestion:
If you prefer to use MySQL or an external server (outside the OpenShift project), modify the `{PRODUCT_INIT}

Reply:
ok